apps/src/code-studio/assets/AddDefaultSprite.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@cdo/apps/templates/Button';
+
+export default class AddDefaultSprite extends React.Component {
+  static propTypes = {
+    onAdd: PropTypes.func.isRequired
+  };
+
+  state = {
+    spriteName: '',
+    spriteCategory: '',
+    displaySuccess: false
+  };
+
+  handleNameChange = event => {
+    this.setState({spriteName: event.target.value, displaySuccess: false});
+  };
+
+  handleCategoryChange = event => {
+    this.setState({spriteCategory: event.target.value, displaySuccess: false});
+  };
+
+  handleSpriteAdd = () => {
+    this.props.onAdd(this.state.spriteName, this.state.spriteCategory);
+    this.setState({spriteName: '', spriteCategory: '', displaySuccess: true});
+  };
+
+  render() {
+    let {displaySuccess} = this.state;
+    return (
+      <div style={styles.assetRow}>
+        <h3>Add a sprite: </h3>
+        <label htmlFor="sprite-path" style={styles.addSpriteLabel}>
+          Sprite name:
+        </label>
+        <input
+          type="text"
+          id="sprite-name"
+          name="name"
+          onChange={this.handleNameChange}
+          required
+        />
+        <label htmlFor="sprite-path" style={styles.addSpriteLabel}>
+          Sprite category path:
+        </label>
+        <input
+          type="text"
+          id="sprite-path"
+          name="path"
+          onChange={this.handleCategoryChange}
+          required
+        />
+        <Button
+          text="Add"
+          color={Button.ButtonColor.orange}
+          onClick={this.handleSpriteAdd}
+        />
+        <i
+          style={{
+            ...styles.checkmark,
+            visibility: displaySuccess ? 'visible' : 'hidden'
+          }}
+          className="fa fa-check"
+          aria-hidden="true"
+        />
+      </div>
+    );
+  }
+}
+
+const styles = {
+  assetRow: {
+    borderTop: '1px solid gray',
```

Comment:
super nit: are there any already-defined color constants we could use here and on line 83? i know we have multiple grays defined, so that one should be easy, but i'm not as sure about 'lightgreen' (i do know we have some greens though)